packages/orchestration/src/exos/chain-account-kit.js

```diff
@@ -100,16 +97,6 @@ export const prepareChainAccountKit = (zone, { watch, asVow }) =>
             'ICA channel creation acknowledgement not yet received.',
           );
         },
-        getBalance(_denom) {
-          // TODO https://github.com/Agoric/agoric-sdk/issues/9610
```

Comment:
re
> no plans to implement them on IcaAccount|ChainAccount

#9610 seems like a plan to implement them. but yes, I suppose it's postponed.

Reply:
Yes there's a plan, but these would be in `CosmosOrchestrationAccount`. `IcaAccount` is concerned with lower level things likes Ports and Connections